tests/openebs-control-plane-validation_test.go

```diff
@@ -0,0 +1,124 @@
+	config, err = clientcmd.BuildConfigFromFlags("", kubeconfig)
+
+	if err != nil {
+		Expect(err).To(BeNil(), "failed to get config")
+	}
+
+	client, err = kubernetes.NewForConfig(config)
+
+	if err != nil {
+		Expect(err).To(BeNil(), "failed to get client")
+	}
+
+	clientSet, err = chaosClient.NewForConfig(config)
+
+	if err != nil {
+		Expect(err).To(BeNil(), "failed to get clientSet")
+	}
+
+	err = v1alpha1.AddToScheme(scheme.Scheme)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+})
+
+//BDD Tests for openebs pool container failure
+var _ = Describe("BDD of openebs pool container failure experiment", func() {
+
+	// BDD TEST CASE 1
+	Context("Check for the components of openebs", func() {
+
+		It("Should check for creation of runner pod", func() {
+
+			
+			//Creating rbac for the experiment
+			By("Creating rbac for the experiment")
+			err = exec.Command("kubectl", "apply", "-f", "https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/openebs/openebs-control-plane-validation/rbac.yaml", "-n", "openebs").Run()
+			Expect(err).To(BeNil(), "fail to create chaos experiment")
+			if err != nil {
+				fmt.Println(err)
+			}
+
+			//Creating Chaos-Experiment
+			By("Creating Experiment")
+			err = exec.Command("kubectl", "apply", "-f", "https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/openebs/openebs-control-plane-validation/rbac.yaml", "-n", "openebs").Run()
+			Expect(err).To(BeNil(), "fail to create chaos experiment")
+			if err != nil {
+				fmt.Println(err)
+			}
+			
+			//Creating Chaos Engine
+			By("Creating Chaos Engine")
+			err = exec.Command("kubectl", "apply", "-f", "https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/openebs/openebs-control-plane-validation/engine.yaml", "-n", "openebs").Run()
+			Expect(err).To(BeNil(), "fail to create chaos engine")
+			if err != nil {
+				fmt.Println(err)
+			}
+
+			time.Sleep(5 * time.Second)
+
+			fmt.Println("Chaosengine created successfully...")
+
+			//Fetching engine-nginx-runner pod
+			runner, err := client.CoreV1().Pods("openebs").Get("control-plane-chaos-runner", metav1.GetOptions{})
+			fmt.Printf("name : %v \n", runner.Name)
+			for i := 0; i < 30; i++ {
+				if string(runner.Status.Phase) != "Succeeded" {
+					time.Sleep(10 * time.Second)
+					runner, _ = client.CoreV1().Pods("openebs").Get("control-plane-chaos-runner", metav1.GetOptions{})
+					fmt.Printf("Current Runner is in %v State, Please Wait ...\n", runner.Status.Phase)
+				} else {
+					break
+				}
+			}
+			Expect(err).To(BeNil())
+			Expect(string(runner.Status.Phase)).To(Or(Equal("Running"), Equal("Succeeded")))
+
+			//Checking the chaosresult
+			By("Checking the chaosresult")
+			app, _ := clientSet.ChaosResults("openebs").Get("control-plane-chaos-openebs-control-plane-validation", metav1.GetOptions{})
+			Expect(string(app.Spec.ExperimentStatus.Verdict)).To(Equal("Pass"), "Verdict is not pass in chaosresult")
```

Comment:
Is the value inside Get function is correct?

Reply:
```=== RUN   TestChaos
Running Suite: BDD test
=======================
Random Seed: 1582659291
Will run 1 of 1 specs

Chaosengine created successfully...
name : control-plane-chaos-runner 
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Running State, Please Wait ...
Currently the Runner pod is in Succeeded State, Please Wait ...
• [SLOW TEST:178.404 seconds]
BDD of openebs control plane chaos experiment
/home/rajdas/go/src/github.com/rajdas98/litmus-e2e/tests/openebs-control-plane-validation_test.go:66
  Check for the components of openebs
  /home/rajdas/go/src/github.com/rajdas98/litmus-e2e/tests/openebs-control-plane-validation_test.go:69
    Should check for creation of runner pod
    /home/rajdas/go/src/github.com/rajdas98/litmus-e2e/tests/openebs-control-plane-validation_test.go:71
------------------------------

Ran 1 of 1 Specs in 178.407 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestChaos (178.41s)
PASS
ok  	command-line-arguments	178.438s
```

Reply:
Yes it is correct